test/integration/content-picker/ContentPicker.e2e.test.js

```diff
@@ -11,51 +13,89 @@ const helpers = {
         cy.getByTestId('be-sub-header').contains('Codepen');
     },
     getRow: rowNum => cy.getByTestId('content-picker').find(`.bcp-item-row-${rowNum}`),
-    selectRow: row =>
-        row
+    selectRow(rowNum) {
+        this.getRow(rowNum)
+            .as('row')
             .click()
             .find('input[type="checkbox"]')
-            .should('be.checked'),
-    unselectRow: row =>
-        row
+            .should('be.checked');
+
+        cy.get('@row').find('.bcp-shared-access-select');
+
+        return cy.get('@row');
+    },
+    unselectRow(rowNum) {
+        this.getRow(rowNum)
+            .as('row')
             .click()
             .find('input[type="checkbox"]')
-            .should('not.be.checked'),
+            .should('not.be.checked');
+
+        cy.get('@row')
+            .find('.bcp-shared-access-select')
+            .should('not.exist');
+    },
 };
 
 describe('ContentPicker', () => {
+    beforeEach(() => {
+        cy.server();
+
+        cy.route(
+            'GET',
+            '**/folders/51964781421?fields=id,name,type,size,parent,extension,permissions,path_collection,modified_at,created_at,modified_by,has_collaborations,is_externally_owned,item_collection&direction=asc&limit=50&offset=0&sort=name',
```

Comment:
Can we avoid hardcoding the file/folder ids? Otherwise it seems like the tests can be pretty brittle if the fixture data changes.